foundry.toml

```diff
@@ -1,6 +1,7 @@
 [profile.default]
 test = 'test/solidity'
 solc_version = '0.8.17'
+evm_version='paris'
```

Comment:
I would suggest that we use london here

Reply:
The default version of Foundry is `paris`, which means you've been using it up until now